packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -72,6 +79,10 @@ class DevToolsAppState extends State<DevToolsApp> {
               EmptyScreen.logging,
               const InfoScreen(),
             ],
+            actions: [
+              HotReloadButton(),
+              HotRestartButton(),
+            ],
```

Comment:
add a bit of padding on the end. these icons look a bit close to the right edge of the view.

Reply:
SGTM

Reply:
Some padding on the right side would also be nice as the icons look a bit close to the right edge of the view.